src/networkextension.cs

```diff
@@ -146,10 +161,26 @@ interface NEAppProxyFlow {
 	
 		[Export ("closeWriteWithError:")]
 		void CloseWrite ([NullAllowed] NSError error);
+
+		[NoWatch, NoTV, NoiOS, Mac (10,15,4)]
+		[Export ("setMetadata:")]
+		void SetMetadata (OS_nw_parameters parameters);
 	
 		[Export ("metaData")]
 		NEFlowMetaData MetaData { get; }
 
+		[NoWatch, NoTV, Mac (10,15,4), iOS (13,4)]
+		[NullAllowed, Export ("networkInterface", ArgumentSemantic.Copy)]
+		OS_nw_interface WeakNetworkInterface { get; set; }
+
+		[NoWatch, NoTV, Mac (10,15,4), iOS (13,4)]
+		NWInterface NetworkInterface {
+			[Wrap ("Runtime.GetINativeObject<NWInterface> (WeakNetworkInterface, true)")]
+			get;
+			[Wrap ("WeakNetworkInterface = value != null ? value.Handle : IntPtr.Zero")]
```

Comment:
the `GetHandle ()` extension method would have done the same ;-)

Reply:
Ahhh I always forget about it heh